dubbo-common/src/main/java/org/apache/dubbo/common/beanutil/JavaBeanDescriptor.java

```diff
@@ -195,15 +186,15 @@ private boolean isValidType(int type) {
         return TYPE_MIN <= type && type <= TYPE_MAX;
     }
 
-    private void notNull(Object obj, String message) {
+    private void notNull(Object obj) {
```

Comment:
why would u change this method,  It seems original method definition is cleaner.

Reply:
consider there is not other case used a different message, so extract to a common method. but maybe u are right.